pyaccel/elements.py

```diff
@@ -525,6 +525,17 @@ def kicktable(self, value):
             raise TypeError('value must be of Kicktable type')
         self.trackcpp_e.kicktable = value._kicktable
 
+    @property
+    def rescale_kicks(self):
+        """Scale factor applied to values given by kicktable.
+        Default value: 0.5"""
+        return self.trackcpp_e.rescale_kicks
```

Comment:
@Gabrielrezende-asc , is not the default value 1.0 ? look at file include/element.h in trackcpp...
one thing is our decision to break a IDs kickmaps into upstream and downstream halves (each having 1/2 of the total kick, using rescale_kick=0.5 for this purpose), another is the default value of the class itself.